awx/main/utils/reload.py

```diff
@@ -48,6 +47,6 @@ def _supervisor_service_command(service_internal_names, command, communicate=Tru
         logger.info('Submitted supervisorctl {} command, not waiting for result'.format(command))
 
 
-def stop_local_services(service_internal_names, communicate=True):
-    logger.warn('Stopping services {} on this node in response to user action'.format(service_internal_names))
-    _supervisor_service_command(service_internal_names, command='stop', communicate=communicate)
+def stop_local_services(communicate=True):
```

Comment:
It turns out we don't actually call `stop_local_services` with explicit service names anymore - the only place we use this is for capacity=0 where stop _everything_, so I simplified it a bit.

@AlanCoding with this change, there's honestly no longer really any need to track `settings.SERVICE_NAME_DICT`.

Reply:
The code where we called this probably never worked with OpenShift to be perfectly honest... plus, that doesn't get called in normal operating procedure, it's only if things have gone sideways. With this celery replacement, we might eventually develop some other method of standing still and not doing anything until a problem is fixed.